app_backend/database.py

```diff
@@ -9,11 +9,12 @@ def __init__(self, username: str, password: str):
 
     def connect(self):
         print("Connecting to database...")
-        if self.client is None and self.username is not None and self.password is not None:
+        if self.username is not None and self.password is not None:
             self.client = MongoClient(f"mongodb+srv://{self.username}:{self.password}@quackcluster.kbete.mongodb.net/myFirstDatabase?retryWrites=true&w=majority")
-            self.db = self.client["QuackCluster"]
+            self.db = self.client["user"]
```

Comment:
Is user going to be our only database or did we have to plan more? Because I honestly don't think we need to do the self.client[user] here and could just have it within the user stuff if we plan to have more databases so we could have separate .py files for each (users, food items, etc)

And I have some new queries for the database that are searching for stuff in the user db, should this all be added here?

Reply:
So, the issue with setting up the client differently is that then we get a little messy about where we put our data. Ultimately, the app backend stores user data so we should just put all user related items in the User collection. 

This includes: name, password, email, friends, liked foods, etc

Reply:
Okay I see. I was confusing this file being in only app_backend and not both dct and app backend. 

And yeah that makes sense, so everything is under the same database but we have different collections for the stuff you mentioned.